src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -761,15 +761,21 @@ export default class ReimbursementRequestService {
    * @param submitter The User marking the request as delivered
    * @param requestId The ID of the reimbursement request to be marked as delivered
    * @param organizationId The organization the user is currently in
+   * @param dateDelivered The date the reimbursed items were delivered
    * @throws NotFoundException if the id is invalid or not there
    * @throws AccessDeniedException if the creator of the request is not the submitter
    * @returns the updated reimbursement request
    */
   static async markReimbursementRequestAsDelivered(
     submitter: User,
     reimbursementRequestId: string,
-    organization: Organization
+    organization: Organization,
+    dateDelivered: Date
   ) {
+    const startOfDay = (date: Date): Date => {
```

Comment:
Maybe move this function to utils so that you can call startOfDay in other files (for example, you use this logic a lot in MarkDeliveredModal.tsx)